complete/src/main/resources/static/app.js

```diff
@@ -230,18 +244,21 @@ function deleteRoom(room) {
 function showError(title, xhr) {
     var serverErrorMessage = xhr.responseJSON == null ? "No response from server." : xhr.responseJSON.message;
     console.error(title + "\n" + serverErrorMessage);
-    var notification = $("<div class=\"toast\" role=\"alert\" aria-live=\"assertive\" aria-atomic=\"true\" style=\"min-width: 30rem\">"
-            + "<div class=\"toast-header bg-danger\">"
-            + "<strong class=\"mr-auto text-dark\">Error</strong>"
-            + "<button type=\"button\" class=\"ml-2 mb-1 close\" data-dismiss=\"toast\" aria-label=\"Close\">"
-            + "<span aria-hidden=\"true\">&times;</span>"
-            + "</button>"
-            + "</div>"
-            + "<div class=\"toast-body\"><p>" + title + "</p><pre><code>" + serverErrorMessage + "</code></pre></div>"
-            + "</div>");
+    var notification = $(`<div class="toast alert alert-warning alert-dismissible fade show" role="alert" role="alert" aria-live="assertive" aria-atomic="true" style="min-width: 30rem">
+    <div class="toast-header bg-danger">
+      <strong class="mr-auto text-dark">Error</strong>
+      <button type="button" class="ml-2 mb-1 close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
+    </div>
+    <div class="toast-body">)
+      `)
+      .append($(`<p/>`).text(title))
+      .append($(`<pre>`)
+        .append($(`<code />`).text(serverErrorMessage))
+      .append(`</pre>
+      </div>`));
     $("#notificationPanel").append(notification);
-    notification.toast({delay: 30000});
-    notification.toast('show');
```

Comment:
Where did these 2 lines go to?

Reply:
Console reported an error: `notification.toast is not a function`. Additionally, notifications were not showing up for me, so I needed to do a rewrite of the notification code.